.travis.yml

```diff
@@ -28,6 +39,13 @@ notifications:
 # command to install dependencies
 install:
   - pip install --upgrade pip
+  - pip install codecov
```

Comment:
This is not correct installation -- this reverts the travis file to an old configuration.